src/sentry/runner/commands/upgrade.py

```diff
@@ -23,8 +23,11 @@ def _check_history():
 
     # If we haven't run all the migration up to the latest squash abort.
     # As we squash more history this should be updated.
-    cursor.execute("SELECT 1 FROM django_migrations WHERE name = '0200_release_indices'")
-    if not cursor.fetchone()[0]:
+    cursor.execute(
+        "SELECT 1 FROM django_migrations WHERE name = '0001_squashed_0200_release_indices'"
```

Comment:
The intent of this check was to fail if the 0200 migration was not run, as the new squash will not run on an installation that has previously run migrations. This change seems to be looking for the new squash instead.